detect_secrets/plugins/aws.py

```diff
@@ -24,13 +24,16 @@ class AWSKeyDetector(RegexBasedDetector):
 
     denylist = (
         re.compile(r'AKIA[0-9A-Z]{16}'),
+        re.compile(r'aws.{0,20}?[\'\"]([0-9a-zA-Z/+]{40})[\'\"]'),
```

Comment:
By default, the RegexBasedDetector plugin selects the first regex group. This regex has been corrected to select the secret value in the first regex group (`([0-9a-zA-Z/+]{40})`). Next, you can see a screenshot of the audit command:
<img width="808" alt="aws_regex" src="https://user-images.githubusercontent.com/69458381/106492044-c9a8ae80-64b7-11eb-97d6-5a8fb4b660d0.png">